specification/Styling/README.md

```diff
@@ -1,32 +1,16 @@
 # Styling
 
-![](figures/example.png)
-
-
-```json
-{
-    "show" : "${Area} > 0",
-    "color" : {
-        "conditions" : [
-            ["${Height} < 60", "color('#13293D')"],
-            ["${Height} < 120", "color('#1B98E0')"],
-            ["true", "color('#E8F1F2', 0.5)"]
-        ]
-    }
-}
-```
-
-Example: Creating a color ramp based on building height.
-
 ## Contributors
 
+_This section is non-normative_
```

Comment:
Are these notes required for the contributors section? In contrast, the editors and acknowledgements in the main specification page do not have this note.

Reply:
The main acknowledgements section is pretty standard. I marked this as non-normative because these would not be part of one single document. I think the best plan would be to add these names to the acknowledgements on the main page then?

Reply:
Ok, yeah that seems good.